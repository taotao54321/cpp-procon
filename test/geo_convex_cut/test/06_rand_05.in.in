52
-98 -9
-97 -16
-96 -21
-93 -33
-90 -41
-87 -47
-82 -54
-75 -63
-58 -80
-53 -83
-44 -88
-37 -91
-26 -95
-12 -98
7 -98
14 -97
29 -94
33 -93
55 -82
60 -78
73 -66
81 -55
86 -46
93 -32
98 -13
98 3
97 11
92 32
91 36
88 44
86 48
77 61
69 70
61 76
51 83
35 92
32 93
25 95
17 97
9 98
0 98
-18 97
-30 93
-42 88
-56 81
-61 77
-69 70
-77 62
-87 45
-91 38
-95 26
-96 21
50
-4 -62 19 49
58 33 -55 -54
18 -97 52 -97
92 -92 -6 -24
8 -22 -5 -39
-90 76 31 54
43 -52 -99 -2
-8 5 43 11
80 -30 -32 12
52 77 -82 70
7 65 64 38
42 50 -97 73
-47 9 52 49
12 -81 -7 64
-72 55 73 -85
-63 -61 31 -74
-3 -93 -23 57
-3 57 73 70
50 45 -74 86
-39 95 -29 -70
-44 -49 59 65
-41 -2 -44 -92
56 -41 -2 -46
45 65 -67 -20
-3 -94 -85 -37
3 93 -46 -48
10 66 88 24
-49 1 -6 -49
80 45 19 -18
-97 -59 65 -38
-64 -88 -27 -41
-8 20 74 90
13 73 -70 72
-63 -53 99 -66
-59 10 -95 -1
28 -25 47 3
7 -54 -77 -12
3 83 92 -42
-57 0 -27 46
78 34 -75 31
52 0 14 -76
-78 52 48 -89
-87 -34 -59 81
69 -28 -75 55
80 57 96 70
-59 95 -39 8
36 57 -25 -34
34 -27 74 -41
-41 -81 46 -34
75 -78 -80 79
