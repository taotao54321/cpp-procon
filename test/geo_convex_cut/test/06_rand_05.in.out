16826.27286460
13294.67819272
30178.16666667
10640.44536287
11258.86731375
4308.55153423
8473.70509684
13910.00607425
15125.51559454
28123.48462302
3825.62970694
25497.63524319
10052.70871840
15397.15225851
17109.74256417
27399.36081753
12143.53900709
4624.56218274
25859.43333022
21461.25292097
15134.48760580
22867.88262911
6496.37639263
19877.79253313
1547.12131806
20452.89311757
3657.23219221
22061.62959787
10029.12677078
23773.96821505
15910.07408123
11129.37562438
27952.67996261
25727.22834494
20311.21053834
22226.87496150
6608.58933002
5615.62460192
6130.09762100
21356.29449615
6321.43548387
19639.33557731
1820.13458502
17124.40857370
16316.95564987
22006.74299806
15455.03969550
17862.26741285
24740.83646813
14862.80880313
